packages/terra-header/docs/README.md

```diff
@@ -0,0 +1,53 @@
+# Terra Header
+
+The terra-header component is a styled layout of a header bar.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-header`
+
+<!-- AUTO-GENERATED-CONTENT:START Peer Dependencies -->
+## Peer Dependencies
+
+<details>
+<summary>View information on this component's peer dependencies.</summary>
+
+This component requires the following peer dependencies be installed in your app for the component to properly function.
+
+| Peer Dependency | Version |
+|-|-|
+| react | ^16.8.5 |
+| react-dom | ^16.8.5 |
+
+
+</details>
+<!-- AUTO-GENERATED-CONTENT:END -->
+
+## Usage
```

Comment:
Can we drop this section in favor of the examples that show source code in the site?

Reply:
Initial attempt at fixing it here https://github.com/cerner/terra-core/pull/2545/commits/b4c876c6792a868e1528f1d9d08f59de74dff3af#diff-fd5e6412906575592c6fd2a2fe3ebce7R35

Also changed the examples to have more differences from each other to more easily tell them apart them.

Reply:
Actually Fixed here now that I understand what meant. https://github.com/cerner/terra-core/pull/2545/commits/a248ac76b9fdc35d7cdd096a27f4d80197edcd7a